### ISSUE TITLE
can't enable mysqlx and group replication  at the same time

### ISSUE THREAD

**Issue by kevinbin** [datacharmer/dbdeployer#48]
**Describe the bug**
Enable mysqlx and group replication at the same time. Failed to start.

The parameters plugin-load are overwritten.

**To Reproduce**

dbdeployer deploy --enable-mysqlx --topology=group replication 5.7 --single-primary -n 6

**plugin-load=group_replication.so**
group_replication=FORCE_PLUS_PERMANENT
group_replication_start_on_boot=OFF
group_replication_bootstrap_group=OFF
transaction_write_set_extraction=XXHASH64
report-host=127.0.0.1
loose-group_replication_group_name="aaaaaaaa-bbbb-cccc-dddd-eeeeeeeeeeee"
loose-group-replication-single-primary-mode=on
gtid_mode=ON
enforce-gtid-consistency
replication crash-safe options
master-info-repository=table
relay-log-info-repository=table
relay-log-recovery=on
loose-group-replication-local-address=127.0.0.1:21048
loose-group-replication-group-seeds=127.0.0.1:21048,127.0.0.1:21049,127.0.0.1:21050,127.0.0.1:21051,127.0.0.1:21052,127.0.0.1:21053

**plugin_load=mysqlx=mysqlx.so**
mysqlx-port=30923
mysqlx-socket=/var/folders/2p/6cyrmdx10b98cwgrpqz2y26r0000gn/T//mysqlx-30923.sock

**Expected behavior**
[Note] Plugin mysqlx reported: 'Scheduler "network" started.'
2019-02-02T11:32:24.211820Z 0 [ERROR] unknown variable 'group_replication=FORCE_PLUS_PERMANENT'
2019-02-02T11:32:24.211836Z 0 [ERROR] Aborting

**Environment:**
 - OS: maxos 10.14
 - dbdeployer version 1.17.0
- mysql 5.7.22

**Comment by weigon** [comment]
--plugin-load=group_replication --plugin-load=mysqlx only loads "mysqlx" as --plugin-load is NOT additive. Use --plugin-load-add instead.

https://dev.mysql.com/doc/refman/8.0/en/server-options.html#option_mysqld_plugin-load-add

**Comment by datacharmer** [comment]
@weigon , thanks for the suggestion.
I found the root cause, and I am considering the options that work across several MySQL versions.

**Comment by weigon** [comment]
--plugin-load-add was added in 5.6.3.

--plugin-load="group_replication;mysqlx" works with 5.5 and later.

**Comment by datacharmer** [comment]
If I only had to deal with MySQLX plugin, that would be enough, but there is also the semi-synch plugins that I need to consider and apply since 5.5.
Luckily, the InnoDB plugin that I used to deal with for MySQL 5.1 is not a necessity anymore. Thus I only need to make sure that the possible plugin combinations are treated correctly from 5.5 to 8.0.

Once users realise that they can load plugins for testing, there are a lot of them that can be tried, and potentially create conflicts.
Either way, don't expect a fix before the upcoming week-end.
